bindings/java/src/main/com/apple/foundationdb/EventKeeper.java

```diff
@@ -167,6 +167,72 @@ default long getTime(Event event, TimeUnit theUnit) {
 			public boolean isTimeEvent() {
 				return true;
 			}
-		};
+		},
+		/**
+		 * The number of single-key get requests.
+		 */
+		GET_REQUESTS,
+		/**
+		 * The number of getKey requests.
+		 */
+		GET_KEY_REQUESTS,
+		/**
+		 * The number of requests for an estimated range size(i.e. {@link
+		 * FDBTransaction#getEstimatedRangeSizeBytes(Range)} or {@link FDBTransaction#getEstimatedRangeSizeBytes(byte[],
+		 * byte[])})
+		 */
+		ESTIMATED_RANGE_SIZE_REQUESTS,
+		/**
+		 * The number of requests for a range split point.
+		 */
+		GET_RANGE_SPLIT_POINT_REQUESTS,
+		/**
+		 * The number of calls to add a conflict range
+		 */
+		ADD_CONFLICT_RANGES,
+		/**
+		 * The number of set calls.
+		 */
+		SETS,
+		/**
+		 * The number of clear requests made.
+		 */
+		CLEAR_REQUESTS,
+		/**
+		 * The number of mutation requests made.
+		 */
+		MUTATIONS,
```

Comment:
Typically we regard sets and clears as mutations, though they do not use the generic `mutate` function. All of the operations available via mutate are collectively known by the name atomic operations, so we could use that.

```suggestion
		ATOMIC_OPS,
```